### PR TITLE
chore: Updating UI on admin settings page for better UX

### PR DESCRIPTION
**Pull request by ankitakinger**
## Description

Updating UI on admin settings page for better UX

#### PR fixes following issue(s)
Fixes [#23835](https://github.com/appsmithorg/appsmith/issues/23835)

#### Type of change
- Bug fix (non-breaking change which fixes an issue)
- Chore (housekeeping or task changes that don't impact user perception)

#### How Has This Been Tested?
- [x] Manual
- [ ] Jest
- [ ] Cypress

#### Dev activity
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] PR is being merged under a feature flag

#### QA activity:
- [ ] [Speedbreak features](https://github.com/appsmithorg/TestSmith/wiki/Test-plan-implementation#speedbreaker-features-to-consider-for-every-change) have been covered
- [ ] Test plan covers all impacted features and [areas of interest](https://github.com/appsmithorg/TestSmith/wiki/Guidelines-for-test-plans/_edit#areas-of-interest)
- [ ] Test plan has been peer reviewed by project stakeholders and other QA members
- [ ] Manually tested functionality on DP
- [ ] We had an implementation alignment call with stakeholders post QA Round 2
- [ ] Cypress test cases have been added and approved by SDET/manual QA
- [ ] Added `Test Plan Approved` label after Cypress tests were reviewed
- [ ] Added `Test Plan Approved` label after JUnit tests were reviewed

### CODE REVIEW

File: app/client/src/ce/constants/messages.ts [hunk 1]
```diff
@@ -1165,6 +1165,16 @@ export const TEST_EMAIL_FAILURE = () => "Sending Test Email Failed";
 export const DISCONNECT_AUTH_ERROR = () =>
   "Cannot disconnect the only connected authentication method.";
 export const MANDATORY_FIELDS_ERROR = () => "Mandatory fields cannot be empty";
+export const FORM_LOGIN_DESC = () =>
+  "Enable your workspace to sign in with Appsmith Form.";
+export const GOOGLE_AUTH_DESC = () =>
+  "Enable your workspace to sign in with Google (OAuth 2.0) single sign-on (SSO).";
+export const GITHUB_AUTH_DESC = () =>
```
Comment (dipyamanbiswas07) [inline app/client/src/ce/constants/messages.ts:1172]: Duplicate?

File: app/client/src/ce/pages/AdminSettings/config/authentication/index.tsx [hunk 2]
```diff
@@ -27,12 +27,20 @@ import {
...
+  GOOGLE_AUTH_DESC,
+  OIDC_AUTH_DESC,
+  SAML_AUTH_DESC,
+  createMessage,
+} from "@appsmith/constants/messages";
 
 const FormAuth: AdminConfigType = {
   type: SettingCategories.FORM_AUTH,
   controlType: SettingTypes.GROUP,
   title: "Form login",
```
Comment (dipyamanbiswas07) [inline app/client/src/ce/pages/AdminSettings/config/authentication/index.tsx:42]: Should this also come from constants?

File: app/client/src/ce/constants/messages.ts (hunk 1 above)
Comment (dipyamanbiswas07) [reply app/client/src/ce/constants/messages.ts:1172]: Ignore this.

File: app/client/src/ce/pages/AdminSettings/config/authentication/index.tsx (hunk 2 above)
Comment (ankitakinger) [reply app/client/src/ce/pages/AdminSettings/config/authentication/index.tsx:42]: not required.

**Review by dipyamanbiswas07 — APPROVED** [review]

**Pull request merged**